app/models/employee.rb

```diff
@@ -0,0 +1,13 @@
+class Employee < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  enum :work_focus, { development: 0, design: 1, business: 2, research: 3 }
```

Comment:
Would probably be better to use [literal enum](https://www.theabstraction.space/p/introducing-literal-enums-in-ruby) but there are time constraints